pycparser/_ast_gen.py

```diff
@@ -187,11 +189,38 @@ def _gen_attr_names(self):
 _PROLOGUE_CODE = r'''
 import sys
 
+def _repr(obj):
+    """
+    Get the representation of an object, with dedicated pprint-like format for lists and tuples.
+    """
+    if isinstance(obj, list):
+        return '[' + (',\n '.join((_repr(e).replace('\n', '\n ') for e in obj))) + '\n]'
```

Comment:
Why the `\n` replacement adding whitespace?

Reply:
This is for adding indentation for parameters, so that they are aligned with their opening delimiter.
The result is similar to the first of [PEP8's indentation examples](https://www.python.org/dev/peps/pep-0008/#id3), each parameter except the first on a new line.

Without it, a `BinaryOp(op='+', left=Constant(type='int', value='1'), right=Constant(type='int', value='2'))` would look like:
```
BinaryOp(op='+', 
         left=Constant(type='int',
         value='1'),
         right=Constant(type='int',
         value='2')
         )
```
instead of:
```
BinaryOp(op='+', 
         left=Constant(type='int',
                       value='1'),
         right=Constant(type='int',
                        value='2')
         )
```